2022/ssm/shared-statistical-methods.md

```diff
@@ -0,0 +1,31 @@
+# Shared Statistical Methods
+
+#### Background
+
+Over the years, initiatives like __CSPA__ (UNECE), __ESSNet Serv2__, __ESSnet I3S__ have all tried to solve the problem of sharing statistical services across countries. Sharing is hard, and there are many hurdles to be crossed. Many of those hurdles are technical, organizational and cultural. 
+
+One area that usually have a lot of focus is the area around _Statistical Methods_. These are being promoted f.ex. through _The awesome list_ (link), and are increasingly released to the public domain through various _Open Source_ licenses. 
```

Comment:
I think the term "Statistical Methods" is confusing. We are actually talking about software, not methods. So it would be more something like "Statistical Methods Implementations" (SMIs?)

Reply:
SMI is good.